BEIMA.Client/cypress/integration/AddDevicePage.spec.js

```diff
@@ -41,12 +42,12 @@ describe("Verify Data in fields is still present when invalid coords exist and A
     cy.visit('http://localhost:3000/addDevice')
     cy.get("#typeDropDown").scrollIntoView().click()
     cy.get("#typeDropDown").children().last().scrollIntoView().click()
-    cy.get('#inputBuilding').scrollIntoView().type("Student Union Building")
```

Comment:
Do we need to replace the testing functionality for buildings?

Reply:
The building and device type dropdown use the same shared component, and we have tests for the device type dropdown, so I think we are okay without replacing the building input tests